docs/services/rocketchat.md

```diff
@@ -0,0 +1,87 @@
+## Configuration
```

Comment:
Would you add Parameters section like as follows, please?

https://github.com/argoproj/notifications-engine/blob/aa774e755de808339b9790c995ce8fd14120f003/docs/services/github.md#L3-L11